tests/ExampleTest.php

```diff
@@ -13,7 +13,5 @@ class ExampleTest extends TestCase
      */
     public function testBasicExample()
     {
-        $this->visit('/')
-             ->see('Laravel 5');
     }
 }
```

Comment:
Please revert this file.
This is dummy test but I'll take care of PHPUnit shortly.